vMenu/menus/VehicleOptions.cs

```diff
@@ -497,6 +504,11 @@ private void CreateMenu()
                         {
                             SetVehicleEngineOn(vehicle.Handle, !vehicle.IsEngineRunning, false, true);
                         }
+                        // Destroy engine
+                        else if (item == destroyEngine)
+                        {
+                            SetVehicleEngineHealth(vehicle.Handle, -4000);
```

Comment:
Setting it to 0 should also kill it afaik, or maybe -1 should be good enough. I haven't tested it completely but let me know. Other than that this PR looks good.